### PR TITLE
Got rid of static/templates/no_search_result.html

### PR DESCRIPTION
**Pull request by veloman-yunkan**
This follow-up to #738 is probably the last step before removing the WIP status from #679.

### CODE REVIEW

File: src/server/internalServer.cpp [hunk 1]
```diff
@@ -447,6 +447,11 @@ std::string noSuchBookErrorMsg(const std::string& bookName)
   return "No such book: " + bookName;
 }
 
+std::string noSearchResultsMsg(const std::string& pattern)
+{
+  return "There is no article with the title <b>\"" + encodeDiples(pattern) + "\"</b> and the fulltext search engine is not available for this content.";
```
Comment (mgautierfr) [inline src/server/internalServer.cpp:452]: With #497, we don't redirect to article titled with `pattern`.
We should rewrite this message with "The fulltext search engine is not available for this content" only.

**Review by mgautierfr — CHANGES_REQUESTED** [review]
We should just change the details message but else we are good.

The question about the md5id in cpp is interesting but this PR is working as it is, and we should discuss about this in #712. So this is not a blocker for this PR.

File: src/server/internalServer.cpp (hunk 1 above)
Comment (veloman-yunkan) [reply src/server/internalServer.cpp]: Done

**Review by mgautierfr — APPROVED** [review]

**Comment by mgautierfr** [comment]
Rebased/fixup on master

**Pull request merged**